src/components/Dashboard/TalkDashboard.vue

```diff
@@ -0,0 +1,226 @@
+const talkDashboardStore = useTalkDashboardStore()
+
+const eventRooms = ref<DashboardEventRoom[]>([])
+const loading = ref(false)
+const fetchEventRoomsInterval = ref<ReturnType<typeof setInterval> | null>(null)
+
+onMounted(async () => {
+	fetchEventRooms()
+	fetchEventRoomsInterval.value = setInterval(fetchEventRooms, FIVE_MINUTES)
+})
+
+onBeforeUnmount(() => {
+	if (fetchEventRoomsInterval.value) {
+		clearInterval(fetchEventRoomsInterval.value)
+	}
+})
+
+const conversationsList = computed(() => store.getters.conversationsList)
+const conversationsInitialised = computed(() => store.getters.conversationsInitialised)
+const filteredConversations = computed(() => conversationsList.value?.filter((conversation : Conversation) => {
+	return filterConversation(conversation, ['mentions'])
+}))
+
+/**
+ * Fetches the event rooms
+ */
+async function fetchEventRooms() {
+	try {
+		loading.value = true
+		eventRooms.value = await talkDashboardStore.fetchDashboardEventRooms() ?? []
+		loading.value = false
+	} catch (error) {
+		console.error('Error fetching event rooms:', error)
+		showError(t('spreed', 'Error fetching event rooms'))
+	}
+}
+
+/**
+ * Creates a new group conversation and navigates to the conversation page.
+ */
+async function startMeeting() {
+	try {
+		const conversation = await store.dispatch('createGroupConversation', {
+			roomName: t('spreed', 'Meeting'), // NOTE: it uses user language statically
+			roomType: CONVERSATION.TYPE.PUBLIC,
+		})
+		router.push({
+			name: 'conversation',
+			params: { token: conversation.token },
+			hash: '#direct-call',
+		})
+	} catch (error) {
+		console.error('Error creating conversation:', error)
+		showError(t('spreed', 'Error creating a meeting'))
+	}
+}
+</script>
+<template>
+	<div class="talk-dashboard-wrapper">
+		<div class="talk-dashboard__header">
+			{{ t('spreed', 'Talk home') }}
+		</div>
+		<div class="talk-dashboard__title">
+			<span class="title">{{ t('spreed', 'Upcoming meetings') }}</span>
+			<NcButton type="primary"
+				@click="startMeeting">
+				<template #icon>
+					<IconVideo />
+				</template>
+				{{ t('spreed', 'Start meeting now') }}
+			</NcButton>
+		</div>
+		<div v-if="!loading && eventRooms.length > 0" class="talk-dashboard__event-cards">
+			<EventCard v-for="eventRoom in eventRooms"
+				:key="eventRoom.eventLink"
+				:event-room="eventRoom"
+				class="talk-dashboard__event-card" />
+		</div>
+		<div v-else-if="loading"
+			class="event-cards__loading-placeholder">
+			<LoadingPlaceholder type="event-cards" />
+		</div>
+		<NcEmptyContent v-else
+			class="talk-dashboard__empty-content"
+			:name="t('spreed', 'No upcoming meetings')"
+			:description="t('spreed', 'You have no upcoming meetings scheduled in the next 7 days')">
+			<template #icon>
+				<IconCalendarBlank :size="40" />
+			</template>
+		</NcEmptyContent>
+		<NcButton class="talk-dashboard__devices-button"
+			type="tertiary"
+			@click="emit('talk:media-settings:show', '')">
```

Comment:
Maybe pass a param here instead of having computed `isInTalkDashboard`?
```suggestion
			@click="emit('talk:media-settings:show', 'device-check')">
```

As a follow-up, I would replace 'Choose device' section in Talk Settings with that button